package.json

```diff
@@ -61,7 +61,7 @@
 	"devDependencies": {
 		"@typescript-eslint/eslint-plugin": "^5.33.0",
 		"@typescript-eslint/parser": "^5.33.0",
-		"eslint": "^8.22.0",
+		"eslint": "8.22.0",
```

Comment:
This isn't really an update, check out the difference between `^`, `~` and without any prefix. 
If you really have to update to the latest version it's `8.23.0`

https://www.npmjs.com/package/eslint
https://www.geeksforgeeks.org/difference-between-tilde-and-caret-in-package-json/